saleor/graphql/product/mutations/products.py

```diff
@@ -110,6 +110,8 @@ class CollectionInput(graphene.InputObjectType):
     description = graphene.String(description='Description of the collection.')
     background_image = Upload(description='Background image file.')
     seo = SeoInput(description='Search engine optimization fields.')
+    published_date = graphene.String(
```

Comment:
Shouldn't this be of the type `Date`?

Reply:
Yeah It should. I got inspiration from PageInput (saleor/graphql/page/mutations.py) available_on field and set string instead of date.Also with this change I think that a change will be required in conftest.py as json module does not serialize date, probably djangoJSONEncoder will do the job. Tomorrow I will push another commit for review with fix.

Reply:
Pushed commit with changes, but I think that djangoJSONEncoder causes security/snyk to fail (I do not have access), there was nothing new except this change, so I think that's the reason. That's probably the reason why PageInput has date as string.

Reply:
Pushed commit with changes related to last review, but I think that djangoJSONEncoder causes security/snyk to fail (I do not have access to check), there was not much new except this change, so I think that's the reason. That's probably the reason why PageInput has date as string.